.travis.yml

```diff
@@ -11,7 +11,7 @@ script:
   - cargo build --release
   - cargo test
   - cargo clippy --all-targets --all-features -- -D warnings
-  - find . -name "*.rs" | xargs rustfmt --check
+  - find . -name "*.rs" | grep -v "pnet_packet" | xargs rustfmt --check
```

Comment:
Why do you need this?

Reply:
There are cargo clippy issues need to be fixed for pnet_packet, but it's out of cloud-hypervisor.

Reply:
Do this instead as a separate commit: `find . -name "*.rs" | grep -v "target/" | xargs rustfmt --check`